wordpress-mobile/WordPress-iOS
# Fix threaded comments source tracking

Ref: #17638

Through discussions with @develric , I realized I got a couple of the threaded comments sources wrong. This fixes `em!

To test:
- Go to Notifications > Likes > detail > notification header. Verify the source is `comment_like_notification`:
`🔵 Tracked: reader_article_comments_opened <blog_id: xxx, post_id: xxx, source: comment_like_notification, subscription_count: xxx>`
- Go to My Site > Activity Log > Comment activity > tap the word `Comment`. Verify the source is `activity_log_detail`:
`🔵 Tracked: reader_article_comments_opened <blog_id: xxx, post_id: xxx, source: activity_log_detail, subscription_count: xxx>`

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
